src/foundry/client/apps/av/av-config.d.mts

```diff
@@ -16,9 +16,23 @@ declare global {
      */
     constructor(object?: AVMaster | undefined, options?: Partial<Options> | undefined);
 
+    /**
+     * @defaultValue
+     * ```typescript
+     * foundry.utils.mergeObject(super.defaultOptions, {
+     *  title: game.i18n.localize("WEBRTC.Title"),
+     *  id: "av-config",
+     *  template: "templates/sidebar/apps/av-config.html",
+     *  popOut: true,
+     *  width: 480,
+     *  height: "auto",
+     *  tabs: [{navSelector: ".tabs", contentSelector: "form", initial: "general"}]
+     * });
+     * ```
+     */
     static override get defaultOptions(): FormApplicationOptions;
 
-    override getData(options: Partial<Options>): MaybePromise<object>;
+    override getData(options: Partial<Options>): MaybePromise<AVConfig.AVConfigData>;
```

Comment:
I'd like input here on whether this should be a `Partial<AVConfig.AVConfigData>` or not. When the case occurs of someone creating a custom FormApplication to replace a build-in one, you may not require the exact same data attributes, typescript would throw an error however if you aren't at least returning all the properties the original returns if its setup like this. 
For my own project I often end up having to change a `getData`'s return type to be `Partial<returntype>`, so wondering what people think what the default should be?

Reply:
I generally like this, it helps define what `super.getData` returns and there's generally no reason _not_ to call super in getData